pkg/vfs/vfsswift/impl_v1.go

```diff
@@ -710,8 +668,9 @@ func (f *swiftFileCreation) Close() (err error) {
 				vfs.PushDiskQuotaAlert(f.fs, true)
 			}
 		} else {
-			// Deleting the object should be secure since we use X-Versions-Location
-			// on the container and the old object should be restored.
+			// TODO(xxx): We previously used X-Versions-Location on the container and
+			// the old object should be restored so deleting the object was secure.
+			// This not the case anymore so maybe we should add some error handling here.
```

Comment:
We really need to fix this (layout v1 and v2) before merging this PR, or we may lose files in production